flink-streaming-java/src/test/java/org/apache/flink/streaming/util/BroadcastOperatorTestHarness.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.util;
+
+import org.apache.flink.api.common.state.BroadcastState;
+import org.apache.flink.api.common.state.MapStateDescriptor;
+import org.apache.flink.streaming.api.operators.TwoInputStreamOperator;
+import org.apache.flink.streaming.api.operators.co.CoBroadcastWithNonKeyedOperator;
+
+/**
+ * A test harness for testing a {@link CoBroadcastWithNonKeyedOperator}.
```

Comment:
I would relax this constraint and maybe say `A test harness for testing a {@link TwoInputStreamOperator} in a broadcast context.`

Reply:
Relaxing this constraint and switching to `TwoInputStreamOperator` would prevent access to the `BroadcastState` via the current method `getBroadcastState()` in the new test harnesses. It seemed to me like this ability would be handy from the user perspective. What do you think?

Reply:
The code below, i.e.
```
twoInputOperator.getOperatorStateBackend().getBroadcastState(stateDescriptor);
```
works just as well on a regular `TwoInputStreamOperator`. Is that what you're referring to?